src/methods/method_overloading/hard/q3/Overload3.java

```diff
@@ -0,0 +1,29 @@
+package methods.method_overloading.hard.q3;
+
+/**
+ * [J] Write a Java Program that can handle the addition of the 2 following
```

Comment:
We should change this question to have a scanner so that we can provide the inputs to the users.

Reply:
Also, change the question text to something more similar to: 
`Write a Java program with a method called sum that takes two integers and returns their sum. Then, overload the sum method so it can also accept two double values.`